improving.html

```diff
@@ -92,7 +92,7 @@
 	  </div>
 
 	<h2 id="intro" class="no-display">Introduction</h2>
-	<p>Do you need to urgently address accessibility in an on-going website development project? This guide provides guidance and pointers to resources to help you address the most critical issues. A more comprehensive guide to help you address accessibility more thoroughly throughout the design and development process is provided in <a href="https://www.w3.org/WAI/impl/Overview">Planning and Managing Web Accessibility</a>.</p>
+	<p>Do you need to urgently address accessibility in an on-going web project? This guide provides guidance and pointers to resources to help you address the most critical issues. A more comprehensive guide to help you address accessibility more thoroughly during the design and development process is provided in <a href="https://www.w3.org/WAI/impl/Overview">Planning and Managing Web Accessibility</a>.</p>
```

Comment:
We use that expression here, for example: https://www.w3.org/WAI/users/involving – I think it fits the context, most people will think about their web project. I come across a significant amount of people that don’t read if a resource talks about sites, because they think it does not apply to web apps. Another term that is often used is _web property_.